lib/experimental/disable-tinymce.php

```diff
@@ -61,14 +61,15 @@ function gutenberg_post_being_edited_requires_classic_block() {
 		return false;
 	}
 
+	$current_post = null;
+
 	// Handle the post editor.
 	if ( ! empty( $_GET['post'] ) && ! empty( $_GET['action'] ) && 'edit' === $_GET['action'] ) {
-		$current_post = get_post( intval( $_GET['post'] ) );
-		if ( ! $current_post || is_wp_error( $current_post ) ) {
-			return false;
-		}
+		$current_post = get_post( absint( $_GET['post'] ) );
+	}
 
-		$content = $current_post->post_content;
+	if ( ! $current_post ) {
```

Comment:
I removed the `WP_Error` check; the [`get_post`](https://developer.wordpress.org/reference/functions/get_post/#return) returns `null` on failure.